src/components/posts/PostList.tsx

```diff
@@ -20,23 +27,24 @@ function PostList() {
           }
         />
       </div>
-      {content
-        .filter((post) => {
-          if (searchValue === "") return post;
-          else if (
+      {contentPost
+        .filter(post => {
+          const display = false
+          if (
+            searchValue === '' ||
             post.content
               .toLocaleLowerCase()
               .includes(searchValue.toLocaleLowerCase())
           ) {
-            return post;
+            return post
```

Comment:
The idea behind using a local variable `display` is to have a single return statement.